test-utils/src/lib.rs

```diff
@@ -296,3 +300,19 @@ pub fn build_expected_hash(values: &[u64]) -> [Felt; 4] {
 
     expected
 }
+
+/// Mutates a seed and generates a word deterministically
+#[cfg(not(target_family = "wasm"))]
+pub fn seeded_word(seed: &mut [u8; 32]) -> Word {
+    *seed = rand::prng_array(*seed);
+    rand::prng_array(*seed)
+}
+
+/// Mutates a seed and generates an element deterministically
+#[cfg(not(target_family = "wasm"))]
+pub fn seeded_element(seed: &mut [u8; 32]) -> Felt {
+    *seed = rand::prng_array(*seed);
+    let bytes = rand::prng_array(*seed);
+    let num = u64::from_le_bytes(bytes);
+    Felt::new(num)
```

Comment:
`prng_array` will make a call to the PRNG for each element, I believe. So, a more efficient way would be to do it like this:
```Rust
let num = rand::prng_array(*seed)[0];
Felt::new(num)
```

Reply:
Makes sense; fixed